### DISCUSSION TITLE
WindowsAppSdk: general error c1010070: Failed to load and parse the manifest. Windows was unable to parse the requested XML data.

### DISCUSSION THREAD

**Discussion by rcohn** [microsoft/WindowsAppSDK#2725] [Q&A]
Hi -

We recently switched over to using WindowsAppSDK 1.1.2 in a self-contained build configuration using Visual Studio 2022. One of the developers on my team ran into this error when trying to build:
`general error c1010070: Failed to load and parse the manifest. Windows was unable to parse the requested XML data.`

I have not yet heard where anybody else on the team (about six developers) has run into the error.

Can anybody give me a pointer of why such an error should come up or where to go for more information on this?  We have updated to the latest version of Visual Studio 2022.

Thanks very much.

**Comment by castorix** [comment]
From Google, some people met this problem when user's folder contained space, for example : [Manifest embedding impossible if user's folder contains spaces...](https://developercommunity.visualstudio.com/t/manifest-embedding-impossible-if-users-folder-cont/225593), although MS people said it was solved...

**Comment by Laim** [comment]
Having the same problem, also have a user account with a space in it.  Which is annoying, since Microsoft don't have a way of changing it on Windows 10 without it breaking things.

**Comment by NicholasChrzan** [comment]
I was having the same problem with building a self contained app and I went through the process of changing my user folder and it now builds properly.
